crates/curp/src/server/storage/db.rs

```diff
@@ -198,20 +206,23 @@ mod tests {
         let storage_cfg = EngineConfig::RocksDB(db_dir.clone());
         {
             let s = DB::<TestCommand>::open(&storage_cfg)?;
-            s.flush_voted_for(1, 222).await?;
-            s.flush_voted_for(3, 111).await?;
+            let (voted_for, entries) = s.recover()?;
```

Comment:
If revover is necessary for the opened DB, why not included it into the `open` function?

Reply:
The db is only recovered in CurpNode, however, the xline server opens the db before pass it to `CurpNode`. If we merge open and recover, it would require `voted_for` and `entries` be passed around.

Reply:
So we'd better to comment somewhere to tell the api users that these two APIs should be used together.

Reply:
Add in 88fc6465ce3df65a371ff0a40e226b5af910eb39